misc/scripts/install-local.sh

```diff
@@ -67,6 +67,10 @@ function checks() {
 		fancy_message error "Package does not contain version"
 		exit 1
 	fi
+	if echo "$gives" | grep -q ",\|\\s"; then
+		fancy_message error "Expected argument \"gives\" to only contain one word but has multiple"
```

Comment:
```suggestion
		fancy_message error "\`gives\` supports only one field"
```

How about this?

Reply:
We don't use backticks anywhere else in the codebase. I think we should keep it how it is but use @wizard-28 last part (`supports only one field`)

Reply:
aight done